metadata-protobuf/proto/Metaprotocol.proto

```diff
@@ -136,4 +136,31 @@ message ChannelOwnerRemarked {
         VideoReactionsPreference video_reactions_preference = 3;
         ModerateComment moderate_comment = 5;
     }
-}
\ No newline at end of file
+}
+
+message CreateVideoCategory {
+    required string name = 1;
+}
+
+message UpdateVideoCategory {
+    required string video_category_id = 1;
+    required string name = 2;
+}
+
+message DeleteVideoCategory {
+    required string video_category_id = 1;
+}
+
+message ModerateVideoCategories {
+    oneof video_category_moderation {
+        CreateVideoCategory create_category = 1;
+        UpdateVideoCategory update_category = 2;
+        DeleteVideoCategory delete_category = 3;
+    }
+}
+
+message WorkerGroupLeadRemarked {
+    oneof worker_group_lead_remarked {
+        ModerateVideoCategories moderate_video_categories = 1;
+    }
+}
```

Comment:
The nested `oneof` message doesn't seem necessary here, we can achieve the purpose by only a single `oneof` message as well, i.e. directly wrapping `*VideoCategory` messages inside `WorkerGroupLeadRemarked`

The message serialized with the schema described in the PR would have a few extra bytes (needed to encode nested `oneof` field) compared to the schema proposed in this comment

Given that the serialized message would be part of on-chain transaction payload, and hence would impact transaction fees, we should  aim for minimum serialized message size
```suggestion
message WorkerGroupLeadRemarked {
    oneof worker_group_lead_remarked {
        CreateVideoCategory create_category = 1;
        UpdateVideoCategory update_category = 2;
        DeleteVideoCategory delete_category = 3;
    }
}
```